pkg/native/serializers/serializer_cdx.go

```diff
@@ -77,10 +79,9 @@ func (s *CDX) Serialize(bom *sbom.Document, _ *native.SerializeOptions, rawopts
 	if doc.SerialNumber == "" || !isValidCycloneDXSerialNumberFormat(doc.SerialNumber) {
 		if opts.GenerateSerialNumber {
 			if bom.Metadata.Id != "" {
-				namespace := uuid.MustParse("5dbcd03c-dd56-4fff-97af-77f89c66eeba")
-				doc.SerialNumber = "urn:uuid:" + uuid.NewSHA1(namespace, []byte(bom.Metadata.Id)).String()
+				doc.SerialNumber = "urn:uuid:" + uuid.NewSHA1(uuid.MustParse(sbom.NamespaceUUID), []byte(bom.Metadata.Id)).String()
```

Comment:
Minor nit, also might want to leverage `GetMetadata().GetId()` here and in the enclosing `if` check

```suggestion
				doc.SerialNumber = uuid.NewSHA1(uuid.MustParse(sbom.NamespaceUUID), []byte(bom.GetMetadata().GetId())).URN()
```